jwala-integration-test/src/test/resources/com/cerner/jwala/ui/selenium/configuration/uploadWebappResource.feature

```diff
@@ -0,0 +1,20 @@
+Feature: Upload Resource - Webbapp Node
+
+  Scenario:Webapp Node
+    Given I logged in
+    And I am in the configuration tab
+    And I created a group with the name "seleniumGroup"
+    And I created a webapp with following parameters:
+      | webappName  | seleniumWebapp |
+      | contextPath | /hello       |
+      | group       | seleniumGroup  |
+    And I am in the resource tab
+    And I expanded component "seleniumGroup"
+    And I expanded component "Web Apps"
+    And I clicked on component "seleniumWebapp"
+    And I clicked on add resource
+    And I fill in the "Deploy Name" field with "hello-world.war"
+    And I fill in the "Deploy Path" field with "C:\ctp\app\instances"
+    And I choose the resource file "hello-world.war"
+    And I click the upload resource dialog ok button
```

Comment:
We might be able to get away without a "when" but we should follow Cucumber scenario best practices and follow Given-When-Then for consistency